runtime/strategies/search-tokens-to-particles.js

```diff
@@ -22,30 +22,34 @@ module.exports = class SearchTokensToParticles extends Strategy {
       this._byToken[name].push(particle);
     }
   }
-  _findParticles(token) {
-    return this._byToken(token) || [];
-  }
   async generate(strategizer) {
     let findParticles = token => this._byToken[token] || [];
     var results = Recipe.over(strategizer.generated, new class extends RecipeWalker {
       onRecipe(recipe) {
-        if (!recipe.isResolved() || !recipe.search || !recipe.search.unresolvedTokens.length) {
+        if (/*!recipe.isResolved() ||*/ !recipe.search || !recipe.search.unresolvedTokens.length) {
```

Comment:
I'll probably need to flip this back again, but I'm going to work on having the search strategy activate when the recipe is resolved OR when the recipe is a terminal case  (did not generate descendants from any other strategies).

Reply:
I thought that if there is search string, we should resolve as much search terms as possible, so having none of the terms resolved makes the recipe not resolved... but then this strategy cannot be applied, so i certainly miss the point here.

I added a TODO to handle this, but I'd like to discuss it whenever we get a chance.